clusters/threat-actor.json

```diff
@@ -10111,13 +10111,17 @@
           "Hong Kong",
           "Malaysia",
           "India",
-          "Taiwan"
+          "Taiwan",
+          "Macao",
+          "Nigeria",
+          " Daggerfly"
```

Comment:
Are you sure about this one? ;-)

Reply:
i want to go too fast and it's the trap :D Thx for the review !